internal/testserver/server.go

```diff
@@ -68,5 +68,5 @@ func NewTestServer(require *require.Assertions,
 			require.NoError(conn.Close())
 		}
 		cancel()
-	}, revision
+	}, ds, revision
```

Comment:
is this only for the deletenamespace calls?

Reply:
Yes, but other callers might need it too